osu.Game.Rulesets.Taiko/Skinning/Legacy/LegacySwellCirclePiece.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Sprites;
+using osu.Game.Skinning;
+using osuTK;
+
+namespace osu.Game.Rulesets.Taiko.Skinning.Legacy
+{
+    internal partial class LegacySwellCirclePiece : Sprite
+    {
+        [BackgroundDependencyLoader]
+        private void load(ISkinSource skin, SkinManager skinManager)
+        {
+            Texture = skin.GetTexture("spinner-warning") ?? skinManager.DefaultClassicSkin.GetTexture("spinner-circle");
```

Comment:
Same as above, `DefaultClassicSkin` should not be directly referenced. Also why is this falling back to  `spinner-circle` and not `spinner-warning`?

Reply:
This is a workaround that I forgot to comment on. The `spinner-warning` file does not exist in the [osu!classic skin resources](https://github.com/ppy/osu-resources/tree/master/osu.Game.Resources/Skins/Legacy) for lazer, which is why I can't fall back to it. I don't know if the file accidentally got missing or if this is expected.

This is why I'm using `spinner-circle`. The texture is almost the same and can function as a replacement for `spinner-warning` when scaled down, with only the border around the spinner not matching exactly.